lib/content-services/src/lib/security/services/security-controls-groups-marks-security.service.ts

```diff
@@ -0,0 +1,335 @@
+    }
+
+    get marksApi(): SecurityMarksApi {
+        return this.securityMark || (this.securityMark = new SecurityMarksApi(this.apiService.getInstance()));
+    }
+
+    get authorityClearanceApi(): AuthorityClearanceApi {
+        return this.authorityClearance || (this.authorityClearance = new AuthorityClearanceApi(this.apiService.getInstance()));
+    }
+
+    get reloadAuthorityClearance(): Subject<void> {
+        return this._reloadAuthorityClearance;
+    }
+
+    /**
+     * Get All security groups
+     *
+     * @param include Additional information about the security group
+     * @param skipCount The number of entities that exist in the collection before those included in this list.
+     * @param maxItems The maximum number of items to return in the list. Default is specified by UserPreferencesService.
+     * @return Promise<SecurityControlsGroupResponse>
+     */
+    getSecurityGroup(
+        skipCount = DEFAULT_SKIP_COUNT,
+        maxItems = this.userPreferencesService.paginationSize,
+        include = DEFAULT_INCLUDE
+    ): Promise<SecurityControlsGroupResponse> {
+        let securityControlsGroupResponse: SecurityControlsGroupResponse;
+        return new Promise((resolve, reject) => {
+            this.groupsApi
+                .getSecurityGroups({
+                    include,
+                    skipCount,
+                    maxItems
+                })
+                .then((response: SecurityGroupPaging) => {
+                    this.groupsPaginatedSource.next(
+                        (securityControlsGroupResponse = {
+                            pagination: response.list.pagination,
+                            entries: response.list.entries.map(
+                                (group: SecurityGroupEntry) => group.entry
+                            )
+                        })
+                    );
+                    resolve(securityControlsGroupResponse);
+                })
+                .catch((error) => reject(error));
+        });
+    }
+
+    /**
+     * Create security group
+     *
+     * @param input securityGroupBody.
+     * @return Observable<SecurityGroupEntry>
+     */
+    createSecurityGroup(
+        input: SecurityGroupBody
+    ): Observable<SecurityGroupEntry> {
+        this.loadingSource.next(true);
+        const payload: SecurityGroupBody = {
+            ...input
+        };
+        const opts = {
+            DEFAULT_INCLUDE
+        };
+        const promise = this.groupsApi.createSecurityGroup(payload, opts);
+
+        return from(promise).pipe(
+            finalize(() => this.loadingSource.next(false))
+        );
+    }
+
+    /**
+     * Create security marks
+     *
+     * @param securityGroupId The key for the security group id.
+     * @param input securityMarkBody[].
+     * @return Promise<any>
+     */
+    createSecurityMarks(
+        securityGroupId: string,
+        input: SecurityMarkBody[]
+    ): Promise<any> {
```

Comment:
Can we use something different than any?

Reply:
done)